### PR TITLE
title update on manage case

### PR DESCRIPTION
**Pull request by praveenrajendran101**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by praveenrajendran101** [comment]
> Not overly keen on this approach, yes it works but means having to inject the TitleService everywhere. Perhaps a better approach would be to use route data and maybe another interceptor/service/component somewhere? This also means the titles are then solely tied to the routes themselves and not individual components. See for an (outdated) example https://ultimatecourses.com/blog/dynamic-page-titles-angular-2-router-events

> This is solely based on the common text in the titles, if the BA's comeback with a different title later on this implementation wont work

**Comment by paddybasi** [comment]
If that ever becomes the case:
1) Implementing it as an interceptor means the common part of the title (e.g. 'HMCTS Manage Cases |') can be abstracted away. Would therefore require less change.
2) In the 'edge case' where we can't dynamically set the title from the route, at this point we can inject the titleService and manually set it

**Comment by paddybasi** [comment]
Also, have a look at app.constants.ts if we don't want to tie the titles to the routes directly

**Comment by praveenrajendran101** [comment]
This is not edge case, case module is a dynamically loaded module. You are mentioning 'HMCTS Manage Cases |' in the root of the solution and the rest in routes of dynamically loaded cases module. Architecturally, it is like text splitting without any base component from root solution implemented in Cases module. Hence, splitting data between modules is no a good idea.

File: src/cases/containers/case-search/case-search.component.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { Observable, combineLatest, Subscription } from 'rxjs';
 import { AppConfig } from '../../../app/services/ccd-config/ccd-case.config';
 import { ActionBindingModel } from '../../../cases/models/create-case-actions.model';
 import { FormGroup } from '@angular/forms';
+import { Title } from '@angular/platform-browser';
```
Comment (paddybasi) [inline src/cases/containers/case-search/case-search.component.ts]: clean up

File: src/cases/containers/case-list/case-list.component.ts [hunk 2]
```diff
@@ -60,7 +61,10 @@ export class CaseListComponent implements OnInit, OnDestroy {
     public store: Store<fromCaseList.State>,
     private appConfig: AppConfig,
     private definitionsService: DefinitionsService,
-  ) { }
+    private titleService: Title
```
Comment (paddybasi) [inline src/cases/containers/case-list/case-list.component.ts]: This still needed here?

File: src/cases/containers/case-home/case-home.component.ts [hunk 3]
```diff
@@ -1,7 +1,10 @@
+import { ExUITitleService } from 'src/app/shared/services/exui-title.service';
 import { Component } from '@angular/core';
 
 @Component({
   selector: 'exui-case-home',
   templateUrl: 'case-home.component.html'
 })
-export class CaseHomeComponent {}
+export class CaseHomeComponent {
```
Comment (paddybasi) [inline src/cases/containers/case-home/case-home.component.ts]: clean up

File: src/cases/containers/case-filter/case-filter.component.ts [hunk 4]
```diff
@@ -4,6 +4,7 @@ import {Store} from '@ngrx/store';
 import * as fromCaseCreate from '../../store/reducers';
 import {ActionBindingModel} from '../../models/create-case-actions.model';
 import * as fromCasesFeature from '../../store';
+import { Title } from '@angular/platform-browser';
```
Comment (paddybasi) [inline src/cases/containers/case-filter/case-filter.component.ts]: clean up

File: src/cases/containers/case-details-home/case-details-home.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import { Title } from '@angular/platform-browser';
```
Comment (paddybasi) [inline src/cases/containers/case-details-home/case-details-home.ts]: clean up

File: src/cases/containers/case-create/case-create.component.ts [hunk 6]
```diff
@@ -5,6 +5,7 @@ import * as fromCases from '../../../cases/store';
 import * as fromRoot from '../../../app/store';
 import {ActionBindingModel} from '../../models/create-case-actions.model';
 import {Subscription} from 'rxjs';
+import { Title } from '@angular/platform-browser';
```
Comment (paddybasi) [inline src/cases/containers/case-create/case-create.component.ts]: clean up

File: src/cases/containers/case-search/case-search.component.ts (hunk 1 above)
Comment (praveenrajendran101) [reply src/cases/containers/case-search/case-search.component.ts]: done

File: src/cases/containers/case-list/case-list.component.ts (hunk 2 above)
Comment (praveenrajendran101) [reply src/cases/containers/case-list/case-list.component.ts]: done

File: src/cases/containers/case-home/case-home.component.ts (hunk 3 above)
Comment (praveenrajendran101) [reply src/cases/containers/case-home/case-home.component.ts]: done

File: src/cases/containers/case-filter/case-filter.component.ts (hunk 4 above)
Comment (praveenrajendran101) [reply src/cases/containers/case-filter/case-filter.component.ts]: done

File: src/cases/containers/case-details-home/case-details-home.ts (hunk 5 above)
Comment (praveenrajendran101) [reply src/cases/containers/case-details-home/case-details-home.ts]: done

File: src/cases/containers/case-create/case-create.component.ts (hunk 6 above)
Comment (praveenrajendran101) [reply src/cases/containers/case-create/case-create.component.ts]: done

**Pull request merged**